src/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj

```diff
@@ -19,21 +19,41 @@
     <Compile Include="SatelliteContractVersionAttributeTests.cs" />
     <Compile Include="MissingSatelliteAssemblyException.cs" />
     <Compile Include="ResourceSetTests.cs" />
-    <EmbeddedResource Include="Resources\TestResx.netstandard17.resx">
+    <_WorkaroundCoreMSBuildResource Include="Resources\TestResx.netstandard17.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>TestResx.netstandard17.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+    </_WorkaroundCoreMSBuildResource>
     <Compile Include="Resources\TestResx.netstandard17.Designer.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>TestResx.netstandard17.resx</DependentUpon>
     </Compile>
-  </ItemGroup>
-  <ItemGroup>
     <EmbeddedResource Include="Resources\TestResx.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>TestResx.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <!--
+  MSBuild on .NET Core doesn't support non-string resources.
```

Comment:
this makes me a little sad :(

Reply:
It makes me a lot more than a little sad....  I guess the reasoning is because these require WinForms APIs that either weren't or still aren't available on .NET Core. Hopefully this can be solved sometime soon in MSBuild.